etc/submit.sh

```diff
@@ -1,4 +1,13 @@
 #!/bin/bash
+# should be a bit nicer than before
+echo "======== WMAgent bootstrap STARTING at $(TZ=GMT date) on $(hostname -f) ========"
+echo "User id : $(id)"
+echo "Local time : $(date)"
+echo "Current system : $(uname -a)"
+echo "Arguments are $@"
+echo "Current hostname: $(hostname -f)"
```

Comment:
Can you update this line to `echo "Current hostname: $(hostname -f) at $GLIDEIN_CMSSite"`?